TECHNOLOGIES.md

```diff
@@ -23,13 +33,27 @@ This is a partially-complete list of technologies used by Digital Publishing.
   * [ElasticSearch](https://www.elastic.co/)
   * [PostgreSQL](https://www.postgresql.org/)
   * [MongoDB](https://www.mongodb.com/)
+  * [MiniMongo](https://www.npmjs.com/package/minimongo)
   * [Neo4j](https://neo4j.com/)
+  * [Neptune](https://aws.amazon.com/neptune/)
   * [Kafka](https://kafka.apache.org/)
+  * [DapperDox](http://dapperdox.io/)
```

Comment:
DapperDox would be better suited under a heading of Documentation tools or something as it isn't a data store.